tasks/git-hooks/go-mod-tidy.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+from invoke.context import Context
+from invoke.exceptions import UnexpectedExit
+
+from ..go import tidy_all
+
+if __name__ == "__main__":
+    # need to tidy all modules due to the replace directives
+    # disable module lookup so that it fails as early as possible
+    # might be possible to do something smarter once https://github.com/golang/go/issues/27005 is fixed
+    os.environ["GOPROXY"] = "off"
+    ctx = Context()
+    try:
+        tidy_all(ctx)
+    except UnexpectedExit:
+        sys.exit(1)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could you print the exception error msg to avoid it being silent ?

Reply:
I should have included an example in the description, actually the error is printed, the only thing which is silenced is the stack trace